src/Pages/Dashboard.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from "react";
+import { Box, Card, Grid, Typography } from "@mui/material";
+import TheTable from "../Components/Table";
+
+export default function Dashboard() {
+	const [totalOperators, setTotalOperators] = useState(0);
+
+	useEffect(() => {
+		const fetchData = async () => {
+			try {
+				const response = await fetch(
+					"https://660549352ca9478ea17ff019.mockapi.io/gateway/api/table-data"
+				);
+				const jsonData = await response.json();
+				// Calculate the total number of unique operators
+				const uniqueOperators = new Set(jsonData.map((item) => item.operator));
+				setTotalOperators(uniqueOperators.size);
+			} catch (error) {
+				console.error("Error fetching data:", error);
+			}
+		};
+
+		fetchData();
+	}, []);
```

Comment:
Why do we need to fetch data here again if it's already being fetched from the `Table.js` file?